src/modules/roc_audio/resampler_updater.cpp

```diff
@@ -96,13 +98,15 @@ bool ResamplerUpdater::update(packet::timestamp_t time) {
         update_time_ += update_interval_;
     }
 
+    const float adjusted_coef = float(sample_rate_coef_ * fe_.freq_coeff());
+
     if (rate_limiter_.allow()) {
         roc_log(LogDebug, "resampler updater: queue_size=%lu fe=%.5f",
-                (unsigned long)queue_size, (double)fe_.freq_coeff());
+                (unsigned long)queue_size, (double)adjusted_coef);
```

Comment:
It would be reasonable to print both freq_coeff and adjusted_coef here.

Reply:
Fixed